src/utils/nativeTokens.ts

```diff
@@ -4,6 +4,9 @@ import { CHAIN_NAME_TO_CHAIN_ID } from 'graphql/data/util'
 
 export function getNativeTokenDBAddress(chain: Chain): string | undefined {
   const pageChainId = CHAIN_NAME_TO_CHAIN_ID[chain]
+  if (pageChainId === undefined) {
+    return undefined
+  }
```

Comment:
why is this not caught by the default case below? is this related to the token lists changes?

Reply:
this actually is, but I was working w/ the tsconfig suggestion @tinaszheng made in the linked slack thread and it flagged this case too, so why not?